enterprise/utils.py

```diff
@@ -5,6 +5,17 @@
 from __future__ import absolute_import, unicode_literals
 
 
+class NotConnectedToEdX(Exception):
+    """
+    Exception to raise when not connected to OpenEdX.
+
+    In general, this exception shouldn't be raised, because this package is
+    designed to be installed directly inside an existing OpenEdX platform.
```

Comment:
Since we're not expecting this exception to be triggered, if it DOES get triggered for some reason we'll need to keep an eye out for it -- should we write a special log entry?